examples/Feeds/UpdateFlightsFeedItemStringAttributeValue.cs

```diff
@@ -306,7 +300,7 @@ public override string Description
             if (attributeIndex == -1)
             {
                 new ArgumentException($"No matching feed attribute for feed item attribute " +
-                    $"value: {newFeedItemAttributeValue}");
+                                      $"value: {newFeedItemAttributeValue}");
```

Comment:
I think there's an indentation issue here. Can't tell for sure.

Reply:
Correct, fixed.